spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializable.scala

```diff
@@ -40,6 +39,21 @@ import scala.reflect.ClassTag
 trait ModuleSerializable extends Loadable with Savable{
 
 
+  private val bigDLVersion = com.intel.analytics.bigdl.BIGDL_VERSION
+
+  // Separate this two methods for reuse in sub-classes
+  protected def checkVersion[T: ClassTag](module : BigDLModule)
+                                         (implicit ev: TensorNumeric[T]) : Unit = {
+    val version = module.getVersion
+    require(version == bigDLVersion, s"bigDL version mismatch," +
```

Comment:
if version > bigDLVersion, throw an exception. if version < bigDLVersion, should we simply throw an exception?